src/app/contact/contact.module.ts

```diff
@@ -17,7 +18,8 @@ import {RouterModule} from "@angular/router";
     FormsModule,
     MatCardModule,
     MatButtonModule,
-    RouterModule
+    RouterModule,
```

Comment:
here when you import ContactRoutingModule it also  imports RoutingModule as well because ContactRoutingModule  exports the RoutingModule whit the routes registered. So, RoutingModule it can be deleted here